terraform/app/variables.tf

```
@@ -0,0 +1,201 @@
+variable "region" {
+  type        = string
+  default     = "eu-west-2"
+  description = "AWS region"
+}
+
+variable "environment_string" {
```

```suggestion
variable "environment" {
```

Do we need `string` in the variable name, as we're not doing this elsewhere it looks a bit odd. If `environment` isn't clear enough we could go with `app_environment` or something like that?